kernel/Dockerfile

```diff
@@ -82,15 +95,16 @@ RUN case $(uname -m) in \
     esac  && \
     cp /config-${KERNEL_SERIES}-$(uname -m) ${KERNEL_DEF_CONF}; \
     if [ -n "${EXTRA}" ]; then \
-        sed -i "s/CONFIG_LOCALVERSION=\"-linuxkit\"/CONFIG_LOCALVERSION=\"-linuxkit${EXTRA}\"/" ${KERNEL_DEF_CONF}; \
-        if [ "${EXTRA}" = "-dbg" ]; then \
-            sed -i 's/CONFIG_PANIC_ON_OOPS=y/# CONFIG_PANIC_ON_OOPS is not set/' ${KERNEL_DEF_CONF}; \
-        fi && \
-        cat /config${EXTRA} >> ${KERNEL_DEF_CONF}; \
+	cat /config-${KERNEL_SERIES}-$(uname -m)${EXTRA} >> ${KERNEL_DEF_CONF}; \
```

Comment:
can we make this conditional based on `config-${KERNEL_SERIES}-$(uname -m)${EXTRA}` existing?

Reply:
You mean if [ -f "/config-${KERNEL_SERIES}-$(uname -m)${EXTRA}" ] ? But even if ${EXTRA} is NULL, "/config-${KERNEL_SERIES}-$(uname -m)${EXTRA}" still exists. Maybe I can rename this file like "/config-${KERNEL_SERIES}${EXTRA}-$(uname -m)". What do you think?

Reply:
but you check `-n "${EXTRA}"` above